neuralpp/symbolic/sympy_expression.py

```diff
@@ -314,3 +315,43 @@ def from_sympy_function_and_general_arguments(sympy_function: sympy.Basic, argum
             with sympy.evaluate(False):
                 sympy_object = sympy_function(*[sympy_argument.sympy_object for sympy_argument in sympy_arguments])
                 return SymPyFunctionApplication(sympy_object, type_dict)
+
+
+def context_to_variable_value_dict_helper(context: FunctionApplication,
```

Comment:
moved from `sympy_interpreter.py`, change the interface from `Dict[sympy.Basic, Any]` to `Dict[str, Any]` which is a unified interface for all contexts.